### ISSUE TITLE
[Auditbeat] Add ignore errors in audit rules

### ISSUE THREAD

**Issue by fixed77** [elastic/beats#15768]
In auditd there is a rule "-i".
This rule will cause auditctl to continue loading rules when it runs across an unsupported field or a rule with a syntax error but exit with success reason code.

It would be nice to add this feature to auditbeat.
Thus, it would be possible to make the same auditbeat settings for different systems.

**Comment by weastur** [comment]
+1

**Comment by jclemons7** [comment]
+1

**Comment by Dominator-3000** [comment]
I really surprised when don't find this function in auditbeat. When you have a large infrastructure with different software that is not unified, it is very difficult to make a single config that is suitable for everything.

**Comment by weastur** [comment]
Up

**Comment by gwsales** [comment]
This is pretty important, how can we get this addressed?

**Comment by fixed77** [comment]
Up

**Comment by chenwei791129** [comment]
Up, i need this option in auditbeat config

**Comment by jkiv** [comment]
One possible workaround is to run `auditd` along side `auditbeat`.

Use your rules file with `auditd` and specify `socket_type: multicast` in the `auditd` module config in your `auditbeat.xml`:

```yaml
# ...
auditbeat.modules:
- module: auditd
  socket_type: multicast
# ...
```

`auditbeat` will ignore any `audit_rule_files` or `audit_rule` entries and will rely on `auditd` for audit log messages. (https://discuss.elastic.co/t/auditbeat-running-with-auditd/144782/2)

**Comment by jamiehynds** [comment]
Thanks to everyone for their upvotes and comments on this Auditbeat enhancement. Happy to report that we've added a new `ignore_errors` config option via [this PR](https://github.com/elastic/beats/pull/36851). The option will be available from Auditbeat 8.12 onwards.
